scanpipe/templates/scanpipe/includes/issue_report_modal.html

```diff
@@ -0,0 +1,75 @@
+<div class="modal" id="modal-report">
+    <div class="modal-background"></div>
+    
+    <div class="modal-card">
+      <header class="modal-card-head">
+        <p class="modal-card-title">Report this issue on github!!!</p>
+        <button class="delete" aria-label="close"></button>
+      </header>
+      <section class="modal-card-body">
+  
+        <div class="field">
+          <div class="control">
+            <label class="label has-text-weight-semibold" for="{{ issue_report.issue_title.id_for_label }}">Issue Title:</label>
+            <div class="control">
+              <input class="input" value="{{ error.model }}" id="title">
+            </div> 
+          </div>
+        </div>
+  
+        <div class="field">
+          <div class="control">
+            <label class="label has-text-weight-semibold" for="{{ issue_report.project_name.id_for_label }}">Project Name:</label>
+            <div class="control">
+              <input class="input" value="{{ project.name }}" id="project">
+            </div>      
+          </div>
+        </div>
+  
+        <div class="field mt-5">
+          <div class="control">
+            <label class="label has-text-weight-semibold" for="{{ issue_report.project_name.id_for_pipeline }}">Pipeline(s) Used:</label>
+            <div class="control">
+              <input class="input" id="pipeline" value="{% for run in project.runs.all %} {{ run.pipeline_name }},{% endfor %}" >
+            </div>      
+          </div>
+        </div>
+  
+        <div class="field mt-5">
+          <label class="label has-text-weight-semibold" for="{{ issue_report.error.id_for_label }}">Error:</label>
+            <div class="control">
+              <input class="input" id="error">
+              <p class="help">Specify a link or Paste the part of the text</p>
+            </div>
+        </div>
+  
+        <div class="field mt-5">
+          <label class="label has-text-weight-semibold" for="{{ issue_report.description.id_for_label }}">Error Description:</label>
+            <div class="control">
+              <input class="input" id="error_desc">          
+            </div>
+        </div>
+        
+      </section>
+      
+        <footer class="modal-card-foot is-flex is-justify-content-space-between">
+          <button class="button has-text-weight-semibold" type="reset">No, Cancel</button>
+          <button class="button is-warning is-no-close" type="submit" onclick="formSubmit()">Yes, Report</button>

```

Comment:
Use plain HTML for this

```
<form action="https://github.com/nexB/scancode.io/issues/new" method="GET">
  <div>
    <label for="labels">Labels</label>
    <input name="labels" id="labels" value="bugs" />
  </div>
  <div>
    <label for="title">Title</label>
    <input name="title" id="title" value="error when running pipeline ..." />
  </div>
  <div>
    <button>Create issue on GitHub</button>
  </div>
</form>
```
when clicking the button, this will automatically generate this GTE URL:

https://github.com/nexB/scancode.io/issues/new?labels=bugs&title=error+when+running+pipeline+...

Reply:
i don't think so that we can directly generate url in html that's why i used js to get data and generate url

Reply:
yeah but i think this method only allowed for one input and here i am taking every thing seperately
![image](https://user-images.githubusercontent.com/67731696/226861876-3de78e0e-595c-4617-89b2-447144ce7215.png)
and url becomes https://github.com/nexB/scancode.io/issues/new?title=issue+while+running+this+pipeline&body=&Project+Name=alpine-httpie&pipelines=+root_filesystem..
which is not a valid

Reply:
and when i use that js function 
![image](https://user-images.githubusercontent.com/67731696/226865951-4fdbb852-ce7e-451b-a4e3-3133c2205257.png)
it is creating issue like this

Reply:
what say @pombredanne